tests/hip/common/common.h

```diff
@@ -116,7 +116,34 @@ hip_test_device_bo {
   T *&get() {
     return _buffer;
   }
+};
+
+// Abstraction of host buffer so we can do automatic buffer dealocation (RAII)
+template<typename T> class
+hip_test_host_bo {
+  T *_buffer;
+
+public:
+  explicit hip_test_host_bo(size_t size, unsigned int flags) : _buffer(nullptr) {
```

Comment:
Not needed in this, doesn't hurt either.
```suggestion
  hip_test_host_bo(size_t size, unsigned int flags) : _buffer(nullptr) {
```